helm/managed/observability/eck/README.md

```diff
@@ -0,0 +1,44 @@
+- Add elastic and opentelemetry helm repos
+-- helm repo add elastic https://helm.elastic.co
+-- helm repo add otel  https://open-telemetry.github.io/opentelemetry-helm-charts
+
+- Install elastic eck operator, **must be labeled elastic-operator**
+-- helm install elastic-operator elastic/eck-operator -n elastic-system --create-namespace
+
+- Fetch dependency helm charts 
+-- From HPCC-Platform/helm/managed/observability/eck/
+-- helm dependency build .
+
+- Install observability chart, **must be labeled eck-apm**
+-- From HPCC-Platform/helm/managed/observability/eck/
+-- helm install eck-apm .
+
+- Instruct HPCC to export traces to recently deployed otel collector
+-- helm upgrade myhpcc HPCC-Platform/helm/hpcc -f HPCC-Platform/helm/managed/observability/eck/otlp-http-collector-k8s.yaml
+
+or 
+
+-- helm install myhpcc HPCC-Platform/helm/hpcc -f HPCC-Platform/helm/managed/observability/eck/otlp-http-collector-k8s.yaml
+
+- Access trace data
+- Access via Kibana
+-- Kibana should be available on https://localhost:5601
+-- Browser might warn about self-signed certificate
+-- use 'elastic' username
+-- Fetch 'elastic' user pass for kibana ui access
+--- kubectl get secret elasticsearch-es-elastic-user -o=jsonpath='{.data.elastic}' | base64 --decode; echo
+-- Once logged on, navigate to Observability->APM->Traces
```

Comment:
I don't have APM under observability. Going to manage integrations I see that APM is installed, but it isn't showing up under Observability.

Reply:
It seems like Observability -> Applications is APM? 
![Screenshot 2025-07-02 at 8 37 28 AM](https://github.com/user-attachments/assets/70985a1d-95ae-4062-9b70-06b7d3fba067)